react-components/src/architecture/concrete/config/StoryBookConfig.ts

```diff
@@ -19,6 +19,7 @@ import { type BaseTool } from '../../base/commands/BaseTool';
 import { ToggleMetricUnitsCommand } from '../../base/concreteCommands/ToggleMetricUnitsCommand';
 import { MeasurementTool } from '../measurements/MeasurementTool';
 import { ClipTool } from '../clipping/ClipTool';
+import { SetSpeedOptionCommand } from '../../base/concreteCommands/SetKeyboardSpeedCommand';
```

Comment:
`SetKeyboardSpeedCommand`  instead of `SetSpeedOptionCommand` ?

Also, one comment about naming: is it not better to not use "Sets" and "Gets" as a prefix of classes? Maybe only "KeyboardSpeedCommand" or something like this.

As in OOP we use getters and setters for methods to manage parameter values, it might be confusing IMO.

Reply:
I agree. Thanks for comment!